.github/ISSUE_TEMPLATE/3-kubewarden-release.yml

```diff
@@ -0,0 +1,34 @@
+name: Kubewarden Release
+title: "Kubewarden <VERSION> release"
+description: Issue with all the items needed to release a new version of Kubewarden
+projects: ["kubewarden/6"]
+labels: ["kind/chore"]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue tracks the release of a new version of Kubewarden. 
+
+  - type: textarea
+    attributes:
+      label: Checklist
+      description: Base checklist for the release process. Please, add or remove items as needed.
+      value: |
+        This issue tracks the release of a new version of Kubewarden. Please follow the checklist below to ensure a smooth release process.
+        - [ ] Bump policy server version in the `Cargo.toml`.
+        - [ ] Policy Server CI is green
+        - [ ] Tag policy server
+        - [ ] Bump kwctl version in the `Cargo.toml`
+        - [ ] Kwctl CI is green
```

Comment:
Is the "CI is green" step really necessary here? We can't merge the Cargo.toml version bump without a green CI, but this should happen automatically since the Cargo bump itself doesn't introduce any changes.